uint8 dev_count
xmsg_device[] dev_list
